<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserClass</class>
 <widget class="QMainWindow" name="UserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="dogsList">
        <property name="font">
         <font>
          <family>Verdana</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="2" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="ageLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Age:</string>
            </property>
            <property name="buddy">
             <cstring>ageEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ageEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="buddy">
             <cstring>nameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="breedEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="breedLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Breed:</string>
            </property>
            <property name="buddy">
             <cstring>breedEdit</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="weightLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Weight:</string>
            </property>
            <property name="buddy">
             <cstring>weightEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="weightEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="photoLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Link:</string>
            </property>
            <property name="buddy">
             <cstring>photoEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="photoEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="adoptButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string> --Adopt--&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="filterButton">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_9">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="breedFilterLabel">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Breed:</string>
              </property>
              <property name="buddy">
               <cstring>breedFilterEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="breedFilterEdit">
              <property name="font">
               <font>
                <family>Verdana</family>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QFormLayout" name="formLayout_8">
            <item row="0" column="0">
             <widget class="QLabel" name="ageFilterLabel">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Age:</string>
              </property>
              <property name="buddy">
               <cstring>ageFilterEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="ageFilterEdit">
              <property name="font">
               <font>
                <family>Verdana</family>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openFileButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="adoptionList">
      <property name="font">
       <font>
        <family>Verdana</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
